samples/NativeLibrary/Class1.cs

```diff
@@ -35,5 +35,22 @@ public static int WriteLine(IntPtr pString)
             }
             return 0;
         }
+
+        [NativeCallable(EntryPoint = "sumstring")]
+        public static IntPtr sumstring(IntPtr first, IntPtr second)
+        {
+            /* Parse strings from the passed pointers */
+            string my1String = Marshal.PtrToStringAnsi(first);
+            string my2String = Marshal.PtrToStringAnsi(second);
+
+            /* Concatenate strings */
+            string sum = my1String + my2String;
+
+            /*Assign pointer of the concatenated string to sumPointer*/
+            IntPtr sumPointer = Marshal.StringToHGlobalAnsi(sum);
+
+            // Return pointer
```

Comment:
Could you please use consistent commenting style (`/*` vs. `//`)? The most common style in the CoreRT codebase is  `// Comment here`.